website/views/shared/tasks/task.jade

```diff
@@ -44,7 +44,7 @@ li(bindonce='list', id='task-{{::task.id}}', ng-repeat='task in obj[list.type+"s
           span.glyphicon.glyphicon-bullhorn(tooltip=env.t('challenge'))
           | &nbsp;
       // delete
-      a(ng-if='!task.challenge.id', ng-click='removeTask(obj[list.type+"s"], $index)', tooltip=env.t('delete'))
```

Comment:
Does anyone know why this was done this way in the first place? `git blame` says this line was last touched by Marcelo Sizer, if that helps.